identity/src/commonTest/kotlin/com/android/identity/util/X509ViewDataKtTest.kt

```diff
@@ -0,0 +1,98 @@
+package com.android.identity.util
+
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class X509ViewDataKtTest {
+
+    @Test
+    fun `bigIntToString should convert a byte array to a string representation in the given base`() {
```

Comment:
I gather that using spaces in function names is a feature of Kotlin Multiplatform and many people use it for tests... I'm not sure I like it much but maybe that's just me. Curious to see what @kdeus and @sorotokin thinks...